README.rst

```diff
@@ -41,6 +41,29 @@ If a Python core developer approved a PR made by anyone and added the "🤖 auto
 it will be automatically merged once all the CI checks pass.
 
 
+Setup Info
+==========
+
+Requires Python 3.6+
+
+```
```

Comment:
This file uses rst instead of MarkDown.  See https://devguide.python.org/documenting/#showing-code-examples

Reply:
@ezio-melotti can I use `shell`? for the codeblock?

```
.. code-block:: shell
```

Reply:
Sorry, I've updated to use `::`.

Reply:
https://pygments.org/docs/lexers/#lexers-for-various-shells `shell` is accepted, otherwise you can use `bash` or `sh` too.  See also the `.. highlight::` directive as suggested in the review.